packages/cli/src/execution-lifecycle/__tests__/restore-binary-data-id.test.ts

```diff
@@ -30,10 +30,10 @@ function getDataId(run: IRun, kind: 'binary' | 'json') {
 
 const binaryDataService = mockInstance(BinaryDataService);
 
-for (const mode of ['filesystem-v2', 's3'] as const) {
```

Comment:
Why this change? Presumably binary data ID restoration is only for `filesystem-v2` and `s3`?